javascript-view-swapping/main.js

```diff
@@ -0,0 +1,28 @@
+const $container = document.querySelector('.tab-container');
+const $tabElements = document.querySelectorAll('.tab');
+const $viewElements = document.querySelectorAll('.view');
+
+$container.addEventListener('click', function (event) {
+  if (event.target.matches('.tab')) {
+    for (let i = 0; i < $tabElements.length; i++) {
+      if ($tabElements[i] === event.target) {
+        $tabElements[i].className = 'tab active';
+      }
+      for (let i = 0; i < $tabElements.length; i++) {
+        if ($tabElements[i] !== event.target) {
+          $tabElements[i].className = 'tab';
+        }
+      }
+    }
```

Comment:
This shouldn't be a separate `for` loop, you should just move line 13, into an `else` statement on line 10, similar to what you're doing below